line/index.js

```diff
@@ -180,7 +172,7 @@ router.put('/:lineId', auth0Middleware(), screenContentMiddleware(), async (req,
     if (error.status === 409) {
       return handleVersionConflict(res, error)
     }
-    res.status(error.status ?? 500).json({ error: error.message })
+    return respondWithError(res, error.status ?? 500, error.message)
```

Comment:
Here it is returned, (see l:104) but only because the respondWithError() wasn't already in use.